source/uwp/Renderer/lib/AdaptiveImageRenderer.cpp

```diff
@@ -666,4 +746,80 @@ namespace AdaptiveCards::Rendering::Uwp
         auto foundSvg = url.find("svg");
         return !(foundSvg == std::string::npos);
     }
+
+    void render_xaml::XamlBuilder::ParseXmlForHeightAndWidth(winrt::XmlDocument const& xmlDoc, winrt::ImageSource const& imageSource) {
+        if (xmlDoc)
+        {
+            auto rootElement = xmlDoc.DocumentElement();
+            auto height = rootElement.GetAttribute(L"height");
+            auto width = rootElement.GetAttribute(L"width");
+
+            // We only need to set height or width, not both (fixes aspect ratio for person style)
+            bool isHeightSet = false;
```

Comment:
Is it necessary only for personal style? What if png set height and width are not the same and it's intended?:)

Reply:
Yes, this seems to only be necessary for personal style.

One of the images I tested with has different height and width, and this works there as well. My guess is that as long as we set one of them, the ratio remains the same.

The only other solution I thought of was passing style to this method as well, but we'd need to pass it through a lot of different methods to get it here.

What do you think @beervoley

Reply:
I don't think we should "fix" the aspect ratio for person style this way:)
what happens if the picture doesn't have the same width&height and we leave it at that? does it render badly?
I don't think it's our responsibility to "fix" aspect ratio in this case:)
how does personal style is being handled with a usual image?

Reply:
For non-svg, we don't have to set the height and width of the imageSource.

Without this change, this is what the following payload renders.

```JSON
{
  "type": "AdaptiveCard",
  "version": "0.5",
  "body": [
    {
      "type": "Image",
      "url": "symbol:adaptive-card-fixed-height.svg",
      "backgroundColor": "#000000",
      "style": "person"
    }
  ]
}
```
<img width="252" alt="image" src="https://user-images.githubusercontent.com/98650930/215172786-c831f27b-6fad-4821-9db6-3a6e861f86ba.png">

I don't love this solution, but I'm not sure what the solution would be from a card author perspective.

I tested with `height > width` and `width > height`, and I didn't see a difference between this and the previous solution.

Reply:
Whether we crop/stretch, widescreen/letterbox, maintain the native resolution or use the display resolution, this should be part of the functional spec. At the very least we can include a DisplayMode (Fit, Stretch, Crop). @JeanRoca - any consideration?

Reply:
@anna-dingler did you check if it negatively affects regular svgs?

Reply:
@beervoley Yes, I've tested with all of the cases listed in this PR (have height and width set) and in my original PR (uses auto)